### PR TITLE
Fix proxy module

### PR DESCRIPTION
**Pull request by vladikoff**
https://github.com/vladikoff/node-geckodriver/issues/88

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -41,7 +41,13 @@ var executable = 'geckodriver';
 var downloadOptions = {}
 var proxy = process.env.HTTPS_PROXY || process.env.HTTP_PROXY || process.env.https_proxy || process.env.http_proxy || null;
 if (proxy !== null) {
-  downloadOptions.agent = new proxyAgent(proxy);
+  downloadOptions.agent = {
+    https: tunnel.httpsOverHttp({
+      proxy: {
+        host: 'localhost'
```
Comment (pillsilly) [inline index.js:47]: sorry but "if proxy is not null then use localhost as the proxy host" ?

File: index.js [hunk 2]
```diff
@@ -41,7 +41,13 @@ var executable = 'geckodriver';
 var downloadOptions = {}
 var proxy = process.env.HTTPS_PROXY || process.env.HTTP_PROXY || process.env.https_proxy || process.env.http_proxy || null;
 if (proxy !== null) {
-  downloadOptions.agent = new proxyAgent(proxy);
+  downloadOptions.agent = {
```
Comment (pillsilly) [inline index.js:44]: Not sure if it is necessary to switch proxy agent to tunnel 
But in my try I simply changed this line and it works.
```
  downloadOptions.agent = { https: new proxyAgent(proxy) };

```

Comment (vladikoff) [reply index.js:44]: Thanks @pillsilly, do you mind sending a PR?

Comment (pillsilly) [reply index.js:44]: > Thanks @pillsilly, do you mind sending a PR?

😃 here it is https://github.com/vladikoff/node-geckodriver/pull/92

**Pull request closed without merging**